en/starter/generator/project_layout_reference.md

```diff
@@ -79,6 +79,17 @@ All scaffolded projects will have the following files included:
 The `README.md` file contains customized documentation about the contents of the generated
 application including selected capabilities and services.
 
+### Swift Metrics
+
+If you select the [embedded metrics dashboard](core_concepts.html#metrics-dashboard-capability) then the following
+file will be included in the scaffolded project:
+
+```
+├── Sources/                          - Swift source code for the project
+│   ├── Application/                  - Swift module for application logic
+│   │   └── Metrics.swift             - Swift code for Swift Metrics logic
```

Comment:
I'm being picky here but if we're being consistent with our other examples we shouldn't have the middle │character on the line with Metrics.swift defined in it.